### DISCUSSION TITLE
How to use both Parcel 2 and Parcel 1 on the same machine

### DISCUSSION THREAD

**Discussion by Dagge1** [parcel-bundler/parcel#7531] [General] [answered]
Since Parcel 2 does not work with Vue 2 projects, how can I handle Vue 2 projects that I have bundled with Parcel 1.12.5 on the same machine. I had to uninstall parcel 2 and re-install parcel-bundler 1 in order to make it work, but now I cannot use Parcel 2 for new Vue 3 projects. Another question, does Parcel 2 still use the command '_parcel watch index.html_' which uses a separate Node server files for development instead of its own. Thanks!

**Comment by mischnic** [comment, marked as answer]
> Since Parcel 2 does not work with Vue 2 projects, how can I handle Vue 2 projects that I have bundled with Parcel 1.12.5 on the same machine. I had to uninstall parcel 2 and re-install parcel-bundler 1 in order to make it work, but now I cannot use Parcel 2 for new Vue 3 projects

This is exactly the reason why we recommend installing Parcel locally in each project. This ensures that you can always build your older projects as well.

> Another question, does Parcel 2 still use the command 'parcel watch index.html' which uses a separate Node server files for development instead of its own

Yes, that should behave the same as with Parcel 1
